src/material-experimental/mdc-form-field/_form-field-standard-appearance.scss

```diff
@@ -0,0 +1,52 @@
+@import '@material/rtl/mixins';
+@import '@material/floating-label/mixins';
+@import '@material/textfield/variables';
+@import '_form-field-variables';
+
+// Styles for the standard appearance in the MDC-based form-field. MDC has styles for
+// full-width text-fields which kind of look like the standard appearance. Though these
+// styles cannot be used because they do not work for inputs with floating labels, or
+// textareas which are intended to be displayed in the standard appearance. We create our
+// own styles for the standard appearance that work for all types of inputs and textareas
+// and are backwards-compatible with the non-MDC based form-field.
+// TODO(devversion): consider proposing these styles to the MDC text-field implementation.
```

Comment:
Yeah the spec doesn't seem to have this anymore, but we'd need it for backwards compatibility I guess (it doesn't require too much CSS) and MDC web seems to still have a form of that appearance (called `full-width` in their case)

Reply:
I discussed with @jelbourn and we decided that we can just deprecate it now and we won't need to support it when this is released

Reply:
Sounds good to me. I was unsure if we could/should go that way since MDC has this `full-width` style that somewhat looks like the standard appearance.. but given that the spec does not have it, it makes sense to just remove it.

Reply:
Yeah, the thinking here is that we'll likely support both component implementations in v10, and then remove the old one in v11. By that time, the legacy and standard appearances will be out of the spec for a decent amount of time

Reply:
Sounds good to me. I'll remove it.

Reply:
Done.